src/com/google/enterprise/adaptor/database/DatabaseAdaptor.java

```diff
@@ -274,9 +274,9 @@ public void init(AdaptorContext context) throws Exception {
     if (!isNullOrEmptyString(cfg.getValue("db.aclSql"))) {
       aclSql = cfg.getValue("db.aclSql");
       log.config("acl sql: " + aclSql); 
-      aclPrincipalDelimiter = cfg.getValue("db.aclPrincipalDelimiter");
-      log.config("aclPrincipalDelimiter: '" + aclPrincipalDelimiter + "'");
     }
+    aclPrincipalDelimiter = cfg.getValue("db.aclPrincipalDelimiter");
```

Comment:
add comment
// we might have delimiter and no aclSql as ACLs might come from singleDocContentSql rather than aclSql

Reply:
Done.